tests/testthat/test-independent-testing-avg_duration.R

```diff
@@ -45,28 +45,37 @@ test_that("if, say, par = AEDECOD, return the average duration per group per AE
   )
 
   tmp <- db |>
-    group_by(group, par) |>
-    summarise(
+    dplyr::group_by(group, par) |>
+    dplyr::summarise(
       avg = mean(dur, na.rm = TRUE),
-      se = sd(dur, na.rm = TRUE) / sqrt(n())
+      se = sd(dur, na.rm = TRUE) / sqrt(dplyr::n()), .groups = "keep"
     ) |>
     tidyr::pivot_wider(
       id_cols = par,
       names_from = group,
       values_from = c("avg", "se")
     )
 
-  avg <- tmp |>
-    select(starts_with("avg")) |>
+  avg <- dplyr::left_join(data.frame(par = unique(r2rtf_adae$AEDECOD)),
```

Comment:
Add in the sorting that we added into the function itself. Should maintain the sort order as seen in the input `par`.